src/components/books.js

```diff
@@ -0,0 +1,53 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook, removeBook } from '../redux/books/books';
+
+import './books.css';
+
+const uuid = require('uuid');
+
+function Books() {
+  const dispatch = useDispatch();
+  const bookArr = useSelector((state) => state.booksReducer);
+  return (
+    <>
+      <h1>Book List</h1>
+      <form onSubmit={(e) => {
+        e.preventDefault();
+        const newBook = {
+          id: uuid.v4(),
+          title: e.target.title.value,
+          author: e.target.author.value,
+          year: e.target.year.value,
+        };
+        dispatch(addBook(newBook));
+      }}
+      >
+        {/* eslint-disable */}
+        <label>
+          Title:
+          <input type="text" name="title" />
+        </label>
+        <label>
+          Author:
+          <input type="text" name="author" />
+        </label>
+        <label>
+          Year:
+          <input type="text" name="year" />
+        </label>
+        <input type="submit" value="Add Book" />
+      </form>
+      {/* eslint-enable */}
```

Comment:
- [ ] Don't disable eslint errors please fix them without disabling them
- [ ] If a user clicks on the add button, the fields have to be empty so that a user should add other information easily.
![data-15](https://user-images.githubusercontent.com/55635977/141990522-f70ca3d7-0dea-4c28-95b9-bb88212733ca.png)

- [ ] Please validate the fields to prevent the user to add empty data.

![data-16](https://user-images.githubusercontent.com/55635977/141990533-8e7d13ca-baaa-44c7-a48b-bfaf23a62451.png)